src/client/pages/Varen/Varen-thema-config.ts

```diff
@@ -37,7 +37,8 @@ const tableConfigBase = {
 export const tableConfig = {
   [listPageParamKind.inProgress]: {
     title: 'Lopende aanvragen',
-    filter: (vergunning: VarenFrontend) => !vergunning.dateEnd,
+    filter: (vergunning: VarenFrontend) =>
+      !vergunning.processed || vergunning.decision !== 'Verleend',
```

Comment:
Iets dat geweigerd wordt is niet een lopende aanvraag. Je kunt hier gewoon `=> vergunning.processed` gebruiken.